windows/active_directory/create_ad_domain.yml

```diff
@@ -1,6 +1,7 @@
 ---
 - name: Create Active Directory domain
-  hosts: "{{ HOSTS | default('windows') }}"
+  hosts: windows
```

Comment:
why remove the ability to pass hosts as a variable?

Reply:
It seemed unnecessary, SAs have been confused on this step as is

Reply:
It's necessary for the flexibility of the project. By hardcoding the group "windows" it requires the user to have a group called windows and that to only be the server(s) that will be domain controllers otherwise change the playbook. the "HOSTS" variable can default to a specific server (eg. win1) or group via the extra vars on the job template

Reply:
I've added it back